packages/usa-range/src/test/sr-callout--preposition.spec.html

```diff
@@ -0,0 +1,14 @@
+<form class="usa-form">
+  <input
+    id="usa-range"
+    class="usa-range"
+    type="range"
+    min="0"
+    max="100"
+    step="10"
+    value="20"
+    role="slider"
+    data-text-unit="por ciento"
```

Comment:
Since this is a separate test, I'm wondering if it makes sense to only test the `data-text-preposition` attribute here.

```suggestion
```